lib/commands/general.js

```diff
@@ -3,6 +3,15 @@ import log from '../logger';
 import { androidHelpers } from 'appium-android-driver';
 import { errors } from 'appium-base-driver';
 
+const DRIVER_ONLY_SETTINGS = [
```

Comment:
I assume this list should not be duplicated here, but rather extended from base driver

Reply:
yeah good idea